tests/integration-tests/tests/storage/test_fsx_lustre.py

```diff
@@ -200,6 +216,19 @@ def get_fsx_fs_id(cluster, region):
     return utils.retrieve_cfn_outputs(fsx_stack, region).get("FileSystemId")
 
 
+def get_storage_type(fsx_fs_id, region):
+    logging.info("Getting StorageType from DescribeFilesystem API.")
+    fsx = boto3.client("fsx", region_name=region)
+    return fsx.describe_file_systems(FileSystemIds=[fsx_fs_id]).get("FileSystems")[0].get("StorageType")
+
+
+def _test_correct_storage_type(storage_type, fsx_fs_id, region):
+    if storage_type == "HDD":
```

Comment:
you can rewrite this with: 
```python
assert_that(get_storage_type(fsx_fs_id, region)).is_equal_to(storage_type)
```

Reply:
and you can remote it to _test_storage_type, to be consistent with other _test functions.

Reply:
I don't know whether we can get storage_type from the remote, the df -h commands doesn't provide that info. I search for ways to do that and didn't found it. So instead I just make the boto3 call to check the storage type

Reply:
Sorry I meant rename, not remote :D 
> you can rename it to _test_storage_type, to be consistent with other _test functions.

Reply:
Done! :D